rqd/rqd/rqnimby.py

```diff
@@ -20,18 +20,43 @@
 from __future__ import print_function
 from __future__ import division
 
+from abc import ABCMeta, abstractmethod
 import os
 import select
 import time
 import signal
 import threading
-import logging as log
+import logging
+import platform
 
 import rqd.rqconstants
 import rqd.rqutil
 
+if platform.system() == 'Windows':
+    try:
+        import pynput
+    except ImportError:
+        pass
```

Comment:
Do we really want to eat this error? IIUC it should be required if `rqd.rqconstants.USE_NIMBY_PYNPUT == True`, maybe add a check for that? Let's at least log something if it fails.

Reply:
Sure thing, I can add a log for the import error.